python/py-threadpoolctl/Portfile

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-threadpoolctl
+version             2.1.0
+platforms           darwin
+license             NULL
```

Comment:
I believe this should be `license BSD` based on:

https://github.com/joblib/threadpoolctl/blob/master/LICENSE

Reply:
Thanks. That’s fixed in master.